virttest/env_process.py

```diff
@@ -1047,6 +1050,14 @@ def preprocess(test, params, env):
                     env["cpu_driver"] = cpu_driver[0]
                     params["cpu_driver"] = env.get("cpu_driver")
 
+    kvm_module_params = params.get("kvm_module_parameters", "DEFAULT")
+    should_modify_kvm_module_params = kvm_module_params != "DEFAULT"
```

Comment:
Is it actually necessary to specify extra value `DEFAULT` here? Having it empty should do the trick...